ORBIT/manager.py

```diff
@@ -1114,6 +1126,61 @@ def installation_capex_per_kw(self):
 
         return capex
 
+    @property
+    def capex_breakdown(self):
+        """Returns CapEx breakdown by category."""
+
+        unique = np.unique(
+            [*self.system_costs.keys(), *self.installation_costs.keys()]
+        )
+        categories = {}
```

Comment:
Will this pull from custom design/install modules?

Reply:
This will pull costs from custom modules but with the way it is coded up right now they would be classified as 'Misc' without some hacking. I had an idea at lunch that may resolve this though.

Reply:
I revised this a bit so that `ProjectManager` pulls the capex category from individual modules so it'll work with custom modules now. 

In custom modules, you can now define the category like this:

```
class CustomModule(InstallPhase):
    
    capex_category = "Substructure"
    expected_config = {...}
    
    def __init__(self, ):
        ...
```

If it doesn't find capex_category it'll print a warning and put the costs in 'Misc.'